diff --git a/Jenkinsfile b/Jenkinsfile
index 300f0ad..a791540 100644
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -12,7 +12,7 @@ pipeline {
     timestamps()
     buildDiscarder(logRotator(numToKeepStr: '10'))
     throttleJobProperty(
-      categories: ['asterisk'],
+      categories: ['asterisk-vanilla'],
       throttleEnabled: true,
       throttleOption: 'category'
     )
@@ -35,22 +35,8 @@ pipeline {
         ]
       }
     }
-    stage('Docker build') {
-      steps {
-        sh "docker build --no-cache -t wazoplatform/${JOB_NAME}:latest ."
-      }
-    }
-    stage('Docker publish') {
-      steps {
-        sh "docker push wazoplatform/${JOB_NAME}:latest"
-      }
-    }
   }
   post {
-    success {
-      build wait: false, job: 'asterisk-to-asterisk-vanilla'
-      build wait: false, job: 'asterisk-to-asterisk-debug'
-    }
     failure {
       emailext to: "${MAIL_RECIPIENTS}", subject: '${DEFAULT_SUBJECT}', body: '${DEFAULT_CONTENT}'
     }
diff --git a/debian/control b/debian/control
index 7541e44..7375af2 100644
--- a/debian/control
+++ b/debian/control
@@ -1,4 +1,4 @@
-Source: asterisk
+Source: asterisk-vanilla
 Priority: optional
 Section: comm
 Maintainer: Wazo Maintainers <dev@wazo.community>
@@ -49,8 +49,11 @@ Build-Depends:
 Standards-Version: 3.9.6
 Homepage: http://www.asterisk.org/
 
-Package: asterisk
+Package: asterisk-vanilla
 Architecture: any
+Replaces: asterisk
+Conflicts: asterisk
+Provides: asterisk-virtual
 Depends: ${misc:Depends}, ${shlibs:Depends}, adduser, libtcmalloc-minimal4, mpg123, wazo-asterisk-config
 Breaks:
  wazo-res-amqp (<< 17.14~),
@@ -83,31 +86,40 @@ Description: Open Source Private Branch Exchange (PBX) - Wazo version
  .
  This software was heavily patched by Debian and Wazo.
 
-Package: asterisk-dev
+Package: asterisk-vanilla-dev
 Architecture: all
 Section: devel
+Replaces: asterisk-dev
+Conflicts: asterisk-dev
+Provides: asterisk-virtual-dev
 Depends: ${misc:Depends}
-Recommends: asterisk
+Recommends: asterisk-vanilla
 Description: Development files for Asterisk
  Asterisk is an Open Source PBX and telephony toolkit.
  .
  This package contains the include files used if you wish to compile a
  package which requires Asterisk's source file headers.
 
-Package: asterisk-doc
+Package: asterisk-vanilla-doc
 Architecture: all
 Section: doc
+Replaces: asterisk-doc
+Conflicts: asterisk-doc
+Provides: asterisk-virtual-doc
 Depends: ${misc:Depends}
-Recommends: asterisk
+Recommends: asterisk-vanilla
 Description: Source code documentation for Asterisk
  Asterisk is an Open Source PBX and telephony toolkit.
  .
  This package contains the documentation for configuring an Asterisk system.
 
-Package: asterisk-sounds-main
+Package: asterisk-vanilla-sounds-main
 Architecture: all
-Enhances: asterisk
+Enhances: asterisk-vanilla
 Section: comm
+Replaces: asterisk-sounds-main
+Conflicts: asterisk-sounds-main
+Provides: asterisk-virtual-sounds-main
 Depends: ${misc:Depends}
 Description: Sound files for Asterisk
  Asterisk is an Open Source PBX and telephony toolkit.
diff --git a/debian/rules b/debian/rules
index 6224da3..4315188 100755
--- a/debian/rules
+++ b/debian/rules
@@ -13,7 +13,7 @@
 # uncomment the next line to create a "MALLOC_DEBUG" version of asterisk
 #MENUSELECT_EXTRA_ARGS := --enable MALLOC_DEBUG
 # uncomment the next 2 lines to create a "debug" version of asterisk
-#MENUSELECT_EXTRA_ARGS := --enable DONT_OPTIMIZE
+MENUSELECT_EXTRA_ARGS := --enable DONT_OPTIMIZE
 #MENUSELECT_EXTRA_ARGS += --enable DEBUG_THREADS
 # ...and the next line to be able to use the asterisk test framework
 #MENUSELECT_EXTRA_ARGS += --enable TEST_FRAMEWORK
@@ -126,7 +126,13 @@ override_dh_auto_install:
 	$(RM) -f $(CURDIR)/debian/tmp/usr/sbin/refcounter
 
 override_dh_strip:
-	dh_strip -a --dbgsym-migration='asterisk-dbg (<< 8:20.1.0-1~wazo2.deb10)'
+	dh_strip -a --dbgsym-migration='asterisk-vanilla-dbg (<< 8:20.1.0-1~wazo2.deb10)'
+
+override_dh_systemd_enable:
+	dh_systemd_enable --name=asterisk
+
+override_dh_installinit:
+	dh_installinit --name=asterisk
 
 .PHONY: get-orig-source
 
diff --git a/debian/source/options b/debian/source/options
index 08c063e..f88efc3 100644
--- a/debian/source/options
+++ b/debian/source/options
@@ -1 +1 @@
-extend-diff-ignore = "(^|/)(\.dockerignore|AUTHORS|Dockerfile|README.md|CHANGELOG.md|LICENSE|integration_tests.*|contribs.*|zuul.yaml|tox.ini|Jenkinsfile)$"
+extend-diff-ignore = "(^|/)(\.dockerignore|AUTHORS|Dockerfile|README.md|CHANGELOG.md|LICENSE|integration_tests.*|contribs.*|zuul.yaml|tox.ini|Jenkinsfile|vanilla.patch)$"
diff --git a/zuul.yaml b/zuul.yaml
index 205e50c..061d1cc 100644
--- a/zuul.yaml
+++ b/zuul.yaml
@@ -3,13 +3,13 @@
       - debian-packaging-bookworm
     wazo-check:
       jobs:
-        - asterisk-tox-integration
+        - asterisk-vanilla-tox-integration
     wazo-gate:
       jobs:
-        - asterisk-tox-integration
+        - asterisk-vanilla-tox-integration
 
 - job:
-    name: asterisk-tox-integration
+    name: asterisk-vanilla-tox-integration
     description: Run asterisk integration tests
     parent: wazo-tox-integration-py311
     vars:
